react-common/components/controls/Input.tsx

```diff
@@ -69,6 +69,10 @@ export const Input = (props: InputProps) => {
 
     let container: HTMLDivElement;
 
+    React.useEffect(() => {
+        setValue(initialValue || "");
+    }, [initialValue]);
```

Comment:
Is this behavior something we should support? I feel like `initialValue` should be something that should not change. Can the fix here be that the playback controls use onChange?

Reply:
`initialValue` probably isn't the best name for this prop, it should probably just be called `value`. i don't think there's any reason to prevent it from changing though. the child component should respect the props passed in from the parent.